doc/source/whatsnew/v3.0.0.rst

```diff
@@ -14,10 +14,108 @@ including other versions of pandas.
 Enhancements
 ~~~~~~~~~~~~
 
-.. _whatsnew_300.enhancements.enhancement1:
+.. _whatsnew_300.enhancements.string_dtype:
 
-Enhancement1
-^^^^^^^^^^^^
+Dedicated string data type by default
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Historically, pandas represented string columns with NumPy ``object`` data type.
+This representation has numerous problems: it is not specific to strings (any
+Python object can be stored in an ``object``-dtype array, not just strings) and
+it is often not very efficient (both performance wise and for memory usage).
+
+Starting with pandas 3.0, a dedicated string data type is enabled by default
+(backed by PyArrow under the hood, if installed, otherwise falling back to
+NumPy). This means that pandas will start inferring columns containing string
```

Comment:
```suggestion
``object``-dtype backed by NumPy). This means that pandas will start inferring columns containing string
```

Reply:
With that edit it seems to suggest that it falls back to "object" dtype (as we currently use)? But it still has a StringDtype, just using numpy instead of pyarrow under the hood. So it is "backed by NumPy object dtype under the hood", so something like:

```suggestion
being backed by NumPy ``object``-dtype). This means that pandas will start inferring columns containing string
```